primitives/runtime/src/testing.rs

```diff
@@ -160,9 +160,18 @@ pub type DigestItem = generic::DigestItem<H256>;
 pub type Digest = generic::Digest<H256>;
 
 /// Block Header
-pub type Header = generic::Header<u64, BlakeTwo256>;
-
-impl Header {
+pub type Header<B> = generic::Header<B, BlakeTwo256>;
+
+/// Trait bounds required by `traits::Header` for block number type.
+pub trait BlockNumberT: traits::Member + traits::MaybeSerializeDeserialize + Debug + hash::Hash +
```

Comment:
I think it might help us reduce code in other places as well, if we move this to traits::headers and export it.